scripts/production/homology_stats_rapid.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script fetches the number of coding genes and number of coding genes with homologies
+against the taxonomically closest reference species.
+
+Usage example::
+    python homology_stats_rapid.py\
+    -d mysql://ensro@mysql-ens-sta-5:4684/accipiter_gentilis_compara_109\
+    -r ensembl_compara_references -o gallus.json
+"""
+
+import argparse
+import json
+from typing import Tuple, List
+
+from sqlalchemy import create_engine, text
+from sqlalchemy.engine import Connection
+from ensembl.database import DBConnection
+from ensembl.ncbi_taxonomy.api.utils import Taxonomy
+
+
+def parse_arguments() -> argparse.Namespace:
+    """
+    Parse command line arguments.
+
+    Returns:
+        argparse.Namespace: Parsed command line arguments
```

Comment:
In this and other docstrings, I'd suggest to remove the type information, as the function annotations already contain type information:
```suggestion
        Parsed command line arguments
```